terraform/deployments/opensearch/main.tf

```diff
@@ -0,0 +1,180 @@
+terraform {
+  cloud {
+    organization = "govuk"
+    workspaces {
+      tags = ["opensearch", "eks", "aws"]
+    }
+  }
+  required_version = "~> 1.5"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+locals {
+  domain            = "${var.service}-engine"
+  custom_domain     = "${local.domain}.${data.aws_route53_zone.opensearch.name}"
+  subnet_ids        = data.terraform_remote_state.infra_networking.outputs.private_subnet_rds_ids
```

Comment:
This one might need some 👀  too. I have used the rds subnet ids: example here: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/opensearch_domain#vpc-based-opensearch